index.js

```diff
@@ -18,12 +18,30 @@ class ElectronPreferences extends EventEmitter2 {
 		super();
 
 		_.defaultsDeep(options, {
+			config: {
+				debounce: 150,
+			},
 			sections: [],
 			webPreferences: {
 				devTools: false,
 			},
 		});
 
+		this.options = options;
+
+		// Legacy: Set config values
+		if (options.css && !options.config.css) {
+
+			this.options.config.css = options.css;
```

Comment:
Thanks for thinking of backwards compatibility.
In this case I would add a deprecation warning, indicating `options.css` will be removed in future versions.
```suggestion
			console.warn("DEPRECATED: css option has been deprecated and will be removed in a future version. It now lives under config.css.");
			this.options.config.css = options.css;
```